tests/operator/operator.go

```diff
@@ -872,13 +872,18 @@ spec:
 			newVirtualMachineInstancesPerNode := 10
 			maxDevicesCommandArgument := fmt.Sprintf("--max-devices %d", newVirtualMachineInstancesPerNode)
 
-			By("Updating KubeVirt Object")
+			By("Patching KubeVirt Object")
 			kv, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
+
+			newVMIPerNodePatch, err := patch.New(
+				patch.WithAdd("/spec/configuration/virtualMachineInstancesPerNode", newVirtualMachineInstancesPerNode)).GeneratePayload()
+			Expect(err).ToNot(HaveOccurred())
+
```

Comment:
Would you mind moving this code to above the patch (line 885)?
If the above `Expect` fails this is redundant, and I also think it'll be clearer this way.
WDYT?